qiskit_nature/problems/__init__.py

```diff
@@ -17,9 +17,25 @@
 .. currentmodule:: qiskit_nature.problems
 
 
+Second-Quantization Problems
+++++++++++++++++++++++++++++
+
 .. autosummary::
    :toctree:
 
    second_quantization
 
+Sampling Problems
+++++++++++++++++++++++++++++
+
+.. autosummary::
+   :toctree:
+
+   sampling
+
 """
+from .sampling import ProteinFoldingProblem
+from .second_quantization import ElectronicStructureProblem
+from .second_quantization import VibrationalStructureProblem
```

Comment:
I think we had decided against exposes these imports on this level. @woodsp-ibm Any thoughts?

Reply:
Can you link any related discussion on that? I am not aware of it.

Reply:
I think the rationale was that when we have a first_quantization ElectronicStructureProblem etc that we now have a name collision at that level since it was expected we would have the same problems there - just different implementation as such.